src/containers/shared/utils.js

```diff
@@ -130,6 +122,15 @@ export const localizeNumber = (num, lang = 'en-US', options = {}) => {
   return new Intl.NumberFormat(lang, config).format(number);
 };
 
+export function formatPrice(number, lang = 'en-US', currency = 'USD', decimals = 4) {
+  return number
+    ? localizeNumber(number.toPrecision(decimals), lang, {
+        style: 'currency',
+        currency,
+      })
+    : undefined;
```

Comment:
FYI - we should never return `undefined` but instead return `null`. Outside of the scope of this PR I think.